experiment/coverage_utils.py

```diff
@@ -269,6 +343,46 @@ def generate_json_summary(coverage_binary,
     return result
 
 
+# pylint: disable=too-many-locals
```

Comment:
Same commment about pylint directives.
But actually, can this function be broken up? i'd rather make changes requested by pylint where it makes sense than disable complaints.